packages/trax/trax.0.3.0/descr

```diff
@@ -0,0 +1 @@
+Trax defines a special exception, which is used to store a trace of where the exception was raised and re-raised, independently from the state of the call stack.
```

Comment:
It's better to give some feedback after your commits, so that people notice something happened.